mlflow/utils/autologging_utils/__init__.py

```diff
@@ -568,3 +568,12 @@ def get_instance_method_first_arg_value(method, call_pos_args, call_kwargs):
             inspect.Parameter.VAR_POSITIONAL,
         ]
         return call_kwargs.get(first_arg_name)
+
+
+def get_instance_method_arg_value(arg_index, arg_name, default_value, call_pos_args, call_kwargs):
```

Comment:
What makes this specific to instance methods? Couldn't this apply to any function?

Reply:
Can we add docs?